articles/precarious-data.md

```diff
@@ -1,51 +1,217 @@
 
+The World Wide Web is precarious. It's a network of information, discussions, data, and analysis that relies on HTTP links to maintain connections between the pieces. Those links are unreliable because the content they point to is unstable. If you rely on an HTTP service as the authoritative source of data you're building on a precarious foundation. Next time you feel the pain of this precarity, I encourage you to take it as a sign that it's time for you to embrace the decentralized web.
 
-The World Wide Web is precarious. Any network of information, discussions, data, or analysis that relies on HTTP links is built on a precarious foundation. Likewise, if you rely on an HTTP service as the authoritative source of data you're building on a precarious foundation. Next time you feel the pain of this precarity, I encourage you to take it as a sign that it's time for you to embrace the decentralized web.
+When you hear about people scrambling to "back up" data from services that are about to be turned off, or when you find yourself backing up that content yourself, I want you to ask “Why are we still using location-based links to point to stuff on the web? Links shouldn't point to locations where the content _might_ be found. They should point to the content itself, which can exist in many locations at once.”
 
-Next time you hear about people scrambling to "back up" data from services that are about to be turned off, I want you to think “Why are we still using location-based links when we point to stuff on the web? Links shouldn't point to locations where the content _might_ be found. They should point to the content itself, which can exist in many locations at once.” 
+Centralization is a disease that the web is suffering from. Information can exist in many places at once, yet we have built the web using links which pretend that the information they're pointing to only exists in one location. This makes locations artificially important, turning them into points of failure and conveying great power and responsibility to whoever controls those locations. That's centralization: Despite the fact that information can exist in many places at once, we link to information as if it existed at singular locations, which forces us to rely on central locations whenever we want to acquire information or traverse links between pages, datasets, documents, etc.
```

Comment:
This is good. I'm worried you lost some readers above by getting to this later - I don't know what your retention rate is for this document (1 minute of confusion? 30 seconds? 2 seconds?).